rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -26,6 +26,4 @@
     name: "{{ item }}"
     state: present
     extra_args: "{{ pip_install_options | default('') }}"
-  with_flattened:
-    - maas_pip_packages
-    - maas_pip_dependencies
+  with_items: maas_pip_packages + maas_pip_dependencies
```

Comment:
I'd be willing to leave this as is and have a follow on patch that changes it to what @sigmavirus24 suggested

    with_flattened:
      - "{{ maas_pip_packages }}"
      - "{{ maas_pip_dependencies }}"